src/app/banner/banner.component.ts

```diff
@@ -9,10 +12,32 @@ import { environment } from '../../environments/environment';
 })
 export class BannerComponent implements OnInit {
   showBanner: boolean;
-  constructor() { }
+  mismatchedVersion = false;
+  versionFromAPI: string;
+  versionBuiltWith: string;
+
+  constructor(private metadataService: MetadataService) { }
 
   ngOnInit() {
     this.showBanner = environment.staging;
+
+    this.metadataService.getMetadata()
+      .subscribe(
+        (metadata: Metadata) => {
+          if (metadata.hasOwnProperty('version')) {
+            this.versionFromAPI = metadata['version'];
+            this.versionBuiltWith = versions.version;
```

Comment:
Can probably move this out of unsubscribe

Reply:
Do you mean out of subscribe? And move it to a function?

Reply:
out of subscribe